models/advanced-home-example/IRoom.json

```diff
@@ -1,33 +1,42 @@
-{
-  "@id": "dtmi:com:adt:dtsample:room;1",
-  "@type": "Interface",
-  "@context": "dtmi:dtdl:context;2",
-  "displayName": "Room",
-  "extends": "dtmi:com:adt:dtsample:core;1",
-  "contents": [
-    {
-      "@type": [
-        "Property",
-        "Temperature"
-      ],
-      "name": "temperature",
-      "unit": "degreeFahrenheit",
-      "schema": "double"
-    },
-    {
-      "@type": [
-        "Property",
-        "Humidity"
-      ],
-      "name": "humidity",
-      "unit": "gramPerCubicMetre",
-      "schema": "double"
-    },
-    {
-      "@type": "Relationship",
-      "@id": "dtmi:com:adt:dtsample:room:rel_has_sensors;1",
-      "name": "rel_has_sensors",
-      "displayName": "Room has sensors"      
-    }
-  ]
-}
\ No newline at end of file
+[
+  {
+    "@id": "dtmi:com:adt:dtsample:room;1",
+    "@type": "Interface",
+    "@context": "dtmi:dtdl:context;2",
+    "displayName": "Room",
+    "extends": "dtmi:com:adt:dtsample:core;1",
+    "contents": [
+      {
+        "@type": ["Property", "Humidity"],
+        "name": "humidity",
+        "unit": "gramPerCubicMetre",
+        "schema": "double"
+      },
+      {
+        "@type": "Component",
+        "name": "thermastat",
+        "schema": "dtmi:com:adt:dtsample:thermastat;1"
+      },
+      {
+        "@type": "Relationship",
+        "@id": "dtmi:com:adt:dtsample:room:rel_has_sensors;1",
+        "name": "rel_has_sensors",
+        "displayName": "Room has sensors"
+      }
+    ]
+  },
+  {
+    "@context": "dtmi:dtdl:context;2",
+    "@id": "dtmi:com:adt:dtsample:thermastat;1",
```

Comment:
It's usually spelled "therm**o**stat." Unless this is intentional, might want to CTRL+F replace